CondTools/Geometry/plugins/PGeometricDetBuilder.cc

```diff
@@ -123,8 +123,7 @@ void PGeometricDetBuilder::beginRun(const edm::Run&, edm::EventSetup const& es)
   }
   std::vector<const GeometricDet*> modules = tracker->deepComponents();
```

Comment:
As far as I can tell, this line is useless. Am I missing something, or can it be deleted?

Reply:
I deleted this line in 0dbe44d